web_src/js/features/imagediff.js

```diff
@@ -116,7 +116,9 @@ export function initImageDiff() {
       initOverlay(createContext($imagesAfter[2], $imagesBefore[2]));
     }
 
-    $container.find('> .image-diff-tabs').removeClass('is-loading');
+    for (const el of Array.from(this.children).filter((child) => child.classList.contains('image-diff-tabs'))) {
+      el.classList.remove('is-loading');
+    }
```

Comment:
Just a question: Does `> .foo` not work in `querySelector`?

Reply:
Yeah, it doesn't work

Reply:
Seems we can do it with the `:scope` pseudo class:

https://stackoverflow.com/a/17206138/808699
https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

> When used within a DOM API call — such as [querySelector()](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector), [querySelectorAll()](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll), [matches()](https://developer.mozilla.org/en-US/docs/Web/API/Element/matches), or [Element.closest()](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest) — :scope matches the element on which the method was called.

Reply:
[`f906716` (#30140)](https://github.com/go-gitea/gitea/pull/30140/commits/f906716c09d432c7743c9f46f10fafdf212716aa)